content/applications/finance/accounting/bank/feeds/bank_synchronization.rst

```diff
@@ -121,8 +116,8 @@ Users who have created a database before December 2020 need to install the new m
 use the new functionalities.
 
 To do so, go to :menuselection:`Apps --> Update Apps List`, remove the default filter in the search
-bar and type: **account_online_synchronization**. You can then click on the *Install* button to
-install the new module.
+bar and type: **account_online_synchronization**. You can then click on the :guilable:`Install`
+button to install the new module.
 
 .. image:: bank_synchronization/online-sync-module.png
```

Comment:
the image is broken, if you've deleted it, then the lines should be removed (also, there's no image with that name in the bank_sync folder, only `online-sync-sync-now-dashboard.png`
So I'd suggest:
```suggestion
.. image:: bank_synchronization/online-sync-sync-now-dashboard.png
```

Reply:
after building the doc page, that image might actually not suit the § x)